abi.go

```diff
@@ -8,7 +8,7 @@ import (
 	"os"
 	"syscall"
 
-	"github.com/cilium/ebpf/internal"
+	"github.com/DataDog/ebpf/internal"
```

Comment:
These paths make it harder to contribute these changes upstream - shouldn't use `replace` in our go.mod file to remove the need for this? 

https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive

(I have very little experience with go modules, so let me know if this isn't considered best practice)

Reply:
Curious to know if you have any thoughts on this @brycekahle

Reply:
I'm not sure if we can use a `replace` to handle the name change only in this repo, or if every user of this library would need a `replace` directive from `cilium/ebpf` to `DataDog/ebpf`.

Reply:
As far as I know, Bryce is right, it won't be inherited 😭 I [found this issue](https://github.com/golang/go/issues/37261) that seems to confirm it.

Reply:
That makes sense - thanks for looking into it!